agent/src/util/device_plugin_service.rs

```diff
@@ -58,12 +63,27 @@ pub struct InstanceInfo {
     pub list_and_watch_message_sender: broadcast::Sender<ListAndWatchMessageKind>,
     /// Instance's `InstanceConnectivityStatus`
     pub connectivity_status: InstanceConnectivityStatus,
+    /// Instance's hash id
+    pub instance_id: String,
+    /// Device that the instance represents.
+    /// Contains information about environment variables and volumes that should be mounted
+    /// into requesting Pods.
+    pub device: Device,
+}
+
+#[derive(Clone, Debug, Default)]
+pub struct InstanceConfig {
```

Comment:
what is the thinking behind InstanceConfig as a name?  configuration and instance have pretty specific meanings in akri and this combines them in a way that isn't super-intuitive to me.  

maybe something like DevicePluginContext?

Reply:
DevicePluginContext is a good name, rename InstanceConfig to DevicePluginContext